apps/condo/domains/billing/utils/serverSchema/index.js

```diff
@@ -57,6 +59,28 @@ async function registerBillingReceipts (context, data) {
     })
 }
 
+/**
+ * Sums all DONE or WITHDRAWN payments for billingReceipt for <organization> with <accountNumber> and <period>
+ * @param context {Object}
+ * @param organizationId {string}
+ * @param accountNumber {string}
+ * @param bic {string}
+ * @param bankAccount {string}
+ * @param period {string}
+ * @return {Promise<*>}
+ */
+const getPaymentsSum = async (context, organizationId, accountNumber, period, bic, bankAccount) => {
+    const payments = await  find('Payment', {
```

Comment:
Does it work on more than 100 payments?

Reply:
yep, find works on entities with more then 100 paymenst